.github/workflows/test-wheel-windows.yml

```diff
@@ -265,6 +271,14 @@ jobs:
         shell: bash --noprofile --norc -xeuo pipefail {0}
         run: run-tests core
 
+      # Run only after cuda.bindings tests, so that all wheels are installed already.
+      - name: Run cuda.pathfinder tests
```

Comment:
"Bug" No.2: cuda.pathfinder tests probably should be run after cuda.bindings but before cuda.core, because cuda.core needs the cudart wheel as a test dependency (it's used by CuPy, not by cuda.core).

In the long run, we should really move cuda.pathfinder to its own test workflow. This kind of execution-order-dependent issues is not obvious.

Reply:
Hm, I made the change locally, but then backed it out again: After removing the 32-bit DLLs (135a37d263802a9053d6539508830a5921d28096), I'd expect the tests to work, and it tests a little bit more.